jwql/website/apps/jwql/templates/instrument.html

```diff
@@ -23,48 +31,71 @@ <h1 style="text-align: center">
 
     	<!-- List all possible tools -->
     	<div class="row">
-    		<div class="col-md-6 text-center">
-    			<!-- Images -->
-    			<h2>Images</h2>
-    			<a role="button" class="btn btn-primary my-2 disabled" href={{ url('jwql:unlooked', args=[inst]) }}>Unlooked Images</a>
-				</br>
-    			<a role="button" class="btn btn-primary my-2 disabled" href="#">Image Anomalies</a>
-				</br>
-    			<a role="button" class="btn btn-primary my-2" href={{ url('jwql:archive', args=[inst]) }}>Archived Images</a>
-				</br>
+			<div class="container">
+				<div class="col-md-20 text-center">
+					<h2>Archive</h2>
+					<i class="fas fa-database fa-10x"></i>
+					<hr>
+					<p>View Archived {{ inst }} Images Here</p>
+					<div class="dropdown">
+						<button class="btn btn-primary dropdown-toggle" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+						  Images
+						</button>
+						<div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+						  <a class="dropdown-item" href={{ url('jwql:unlooked', args=[inst]) }}>Unlooked Images</a>
+						  <a class="dropdown-item" href="#">Image Anomalies</a>
+						  <a class="dropdown-item" href={{ url('jwql:archive', args=[inst]) }}>Archived Images</a>
+						</div>
+					</div>
+				</div>
 			</div>
 
-			<div class="col-md-6 text-center">
-				<!-- Monitors -->
-    			<h2>Monitors</h2>
-
-    			{% for key, inst_tools in tools.items() %}
-					{% if key == inst.lower() %}
-						{% for monitor_name, monitor_url in inst_tools %}
-							{% if monitor_url == '#' %}
-								<a role="button" class="btn btn-primary my-2 disabled" href="#">{{ monitor_name }}</a>
-								</br>
-							{% else %}
-								<a role="button" class="btn btn-primary my-2" href="{{ monitor_url }}">{{ monitor_name }}</a>
-								</br>
-							{% endif %}
-						{% endfor %}
-					{% endif %}
-				{% endfor %}
+			<div class="container">
+				<div class="col-md-20 text-center">
+					<!-- Monitors -->
+					<h2>Monitors</h2>
+					<i class="far fa-chart-bar fa-10x"></i>
+					<hr>
+					<p>Explore the {{ inst }} instrument monitor suite.</p>
+					<div class="dropdown">
+						<button class="btn btn-primary dropdown-toggle" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+						  Monitors
+						</button>
+						<div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+							{% for key, inst_tools in tools.items() %}
+								{% if key == inst.lower() %}
+									{% for monitor_name, monitor_url in inst_tools %}
+										{% if monitor_url == '#' %}
+											<a class="dropdown-item" href="#">{{ monitor_name }} (Currently Unavailable)</a>
```

Comment:
This is helpful to mark as unavailable! It looks like some of the archive sections are also unavailable. Could you add something similar there (~line 44)?

Reply:
Thanks @tnking97 I will do it now!